docs/01-get-started.md

```diff
@@ -167,9 +167,9 @@ To learn more about endpoints, see the [Working with endpoints](concepts/working
 
 Serverpod makes it easy to generate serializable classes that can be passed between server and client or used to communicate with the database.
 
-The structure for your serialized classes is defined in yaml-files in the `lib/src/models` directory. Run `serverpod generate` in the home directory of the server to build the Dart code for the classes and make them accessible to both the server and client.
+The structure for your serialized classes is defined in `spy.yaml` files anywhere in the `lib` directory. Run `serverpod generate` in the home directory of the server to build the Dart code for the classes and make them accessible to both the server and client.
```

Comment:
Question: thinking if the standard when referring to extensions is to include leading dot like this `.spy.yaml` (applies everywhere)

Reply:
[73d3ac2](https://github.com/serverpod/serverpod_docs/pull/226/commits/73d3ac21156491c785a46f59ca7ad32a31901743)